docs/changelogs/v0.21.md

```diff
@@ -59,7 +60,7 @@ third-party CIDs.
 #### `client/rpc` migration of `go-ipfs-http-client`
 
 The [`go-ipfs-http-client`](https://github.com/ipfs/go-ipfs-http-client) RPC has
-been migrated into [`client/rpc`](./client/rpc).
+been migrated into [`kubo/client/rpc`](https://github.com/ipfs/kubo/tree/master/client/rpc).
```

Comment:
```suggestion
been migrated into [`kubo/client/rpc`](../client/rpc).
```
mb

Reply:
@Jorropo  `/docs/client/rpc`  does not seem to exist?

Reply:
`../..`